index.html

```diff
@@ -3,7 +3,10 @@
 title: Home
 ---
 
-<h1 class="page-heading">EIPs <a href="https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge"><img src="https://badges.gitter.im/Join%20Chat.svg" alt="Gitter"></a></h1>
+<h1 class="page-heading">EIPs
+  <a href="https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge"><img src="https://badges.gitter.im/Join%20Chat.svg" alt="Gitter"></a>
```

Comment:
While you're editing this, would you mind replacing the `&` with `&amp;` here?

Reply:
Corrected in f6a321c9782995ca067132699aa5be5c2d56f641